checks/process.go

```diff
@@ -187,6 +188,23 @@ func formatCommand(fp *process.FilledProcess) *model.Command {
 	}
 }
 
+func formatIO(fp *process.FilledProcess, lastIO *process.IOCountersStat, before time.Time) *model.IOStat {
+	diff := time.Now().Unix() - before.Unix()
```

Comment:
you can safe this duplication by using the `calculateRate` helper function as we do elsewhere. then you can just put everything inline and have a nice small function.
```
return &model.IOStat{
    ReadRate: calculateRate(fp.IOState.ReadCount, lastIO,ReadCount, before),
    ...
}